dts/common/nordic/nrf54h20.dtsi

```diff
@@ -429,6 +486,34 @@
 				global-domain-id = <12>;
 			};
 
+			cpuflpr_vpr: vpr@8d4000 {
+				compatible = "nordic,nrf-vpr-coprocessor";
+				reg = <0x8d4000 0x1000>;
+				status = "disabled";
+				cpu = <14>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+				ranges = <0x0 0x8d4000 0x4000>;
+
+				cpuflpr_vevif_tx: mailbox@0 {
+					compatible = "nordic,nrf-vevif-task-tx";
+					reg = <0x0 0x1000>;
+					status = "disabled";
+					#mbox-cells = <1>;
+					nordic,tasks = <32>;
+					nordic,tasks-mask = <0xffff0000>;
+				};
+
+				cpuflpr_clic: interrupt-controller@1000 {
```

Comment:
CLIC should be at `0xF0000000` for all VPRs, like in L15:
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/common/nordic/nrf54l15.dtsi#L148
Then assertion in `soc/nordic/validate_base_addresses.c` should pass.

Reply:
IIRC all CLICs are visible from eg app core, so they can't share the same address. Maybe `0xF0000000` is the address as seen from each VPR instance? @hubertmis

Reply:
It seems like there is a misunderstanding here. Looking at the Product Specification documentation, I see that the CLIC registers can be instantiated at different addresses for each VPR instance. In nRF54H20, all of them are at 0xf0000000. This address is available only from a VPR CPU. I don't see any way for a VPR's owner to access CLIC registers.

The address 0x00001000 is a default address for a CLIC instance (default MCLICBASE value), but it is overwritten for each VPR instance in the system.

Reply:
And I can't find anything at address 0x8d5000 (0x8d4000 + 0x1000 offset)